pkg/front/fileGenerator.go

```diff
@@ -0,0 +1,260 @@
+//
```

Comment:
Why do we need this file ?

Reply:
@gregLibert In order to generate HTML files on running the app

In fact, in dev mode, each time I run the `go run main.go`, it generates the html, css & js files through this : 
https://github.com/massalabs/thyra/blob/95b2468a9872a2c1e6947bd615cf0bb381007b2b/cmd/thyra-server/main.go#L19